app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -97,3 +98,5 @@ class HomeViewModel @Inject constructor(
         const val MY_USER_PROFILE_SUBROUTE = "myUserProfile"
     }
 }
+
+data class AvatarData(val asset: UserAvatarAsset? = null, val status: UserAvailabilityStatus = UserAvailabilityStatus.NONE)
```

Comment:
We should extract this to its own separate directory

Reply:
`UserAvailabilityStatus` is really avatar related? Maybe it would be more accurate to rename it to `SelfUserData` ?